docs/content/users/quickstart.md

```diff
@@ -427,8 +427,8 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     ddev get ddev/ddev-elasticsearch
     ddev start
 
-    # if the Magento auth is not found in the Composer global config, save it at the project level
-    if ! ddev exec "composer config --list --global 2>/dev/null | grep -q repo.magento.com" 2>/dev/null; then
+    # if the Magento auth is not found in the Composer config, save it at the project level
+    if ! ddev exec "composer config --list 2>/dev/null | grep -q repo.magento.com || composer config --list --global 2>/dev/null | grep -q repo.magento.com" 2>/dev/null; then
```

Comment:
I might be wrong, but I don't think that `composer config --list --global` is relevant here. The difference is between *DDEV*'s global and local homeadditions, both of which land in the *user* directory inside the project, `~/.composer/auth.json` - So maybe my comment was naive in the first place. Anyway, nothing we do makes it into composer's global config.

Reply:
> So maybe my comment was naive in the first place.

Your suggestion was correct because someone could override `COMPOSER_HOME` in the DDEV's global config.

> both of which land in the user directory inside the project, `~/.composer/auth.json`

Not both, when you have `COMPOSER_HOME` set, it will use a different path.

> I don't think that `composer config --list --global` is relevant here.

It's relevant because it runs after `ddev start`, so `~/.ddev/homeadditions/.composer/auth.json` is in `~/.composer/auth.json` at this point.

It just needs some manual testing for a better understanding.

Reply:
I reverted changes for `composer config --list` as it only shows the config for `composer.json` in the current directory.

On the other hand, `composer config --list --global` shows the config you have in `~/.composer/auth.json` or `COMPOSER_HOME` inside the container.

Reply:
I was confused, of course. `global` config in composer is (normally) ~/.composer, and regular is in the project composer.json.